src/go/configinfo/service_info.go

```diff
@@ -136,21 +137,27 @@ func NewServiceInfoFromServiceConfig(serviceConfig *confpb.Service, id string, o
 }
 
 func (s *ServiceInfo) buildCatchAllBackend() error {
-	protocol, tls, err := util.ParseBackendProtocol(s.Options.BackendProtocol)
+
+	scheme, hostname, port, _, err := util.ParseURI(s.Options.BackendAddress)
+	if err != nil {
+		return fmt.Errorf("error parsing backend uri: %v", err)
+	}
+
+	protocol, tls, err := util.ParseScheme(scheme)
 	if err != nil {
 		return err
 	}
 	if protocol == util.GRPC {
-		s.BackendIsGrpc = true
+		s.AnyBackendIsGrpc = true
 	}
 
 	s.CatchAllBackend = &BackendRoutingCluster{
 		UseTLS:       tls,
 		Protocol:     protocol,
-		HttpProtocol: util.HTTP1, // TODO: default to HTTP1 for now
+		HttpProtocol: util.HTTP1, // TODO(b/149050012): default to HTTP1 for now
```

Comment:
TOTT-486:
> Prefer specifying a Buganizer issue over a username. A well ­written Buganizer issue provides a permanent history that won’t change teams, leave Google, or forget. It also adds additional context by including a full description, related CLs, and status updates.

I've been following this approach for a while, but let me know if we should switch since we are open-source now. I would prefer to keep using bug ids.

Reply:
In github, external users cannot see what is b/xxxxx. 
For yourself, you can track your work in buganizer.

Reply:
Done. In the future, let's only use bug numbers or github issue numbers if there is no assignee for a task.